elementary/monitor/dbt_project/models/tests_recommendation/table_timestamp_columns.sql

```diff
@@ -76,6 +76,18 @@ with
         where loaded_at_field is not null
     ),
 
+    -- Users can provide the timestamp columns for their models,
+    -- if provided, we assign a confidence score of 0 (certain).
+    model_provided_timestamp_columns as (
+        select
+            lower(database_name) as database_name,
+            lower(schema_name) as schema_name,
+            lower(name) as table_name,
+            lower(bigquery_partition_by_field) as column_name
+        from {{ ref("elementary", "dbt_models") }}
+        where bigquery_partition_by_field is not null
```

Comment:
Note that BQ partitions may be on columns that are not timestamps (e.g. [here](https://cloud.google.com/bigquery/docs/partitioned-tables#integer_range))
so I think checking the type is also important.
(related to what I wrote in the dbt PR - I think it's good to save the entire `partition_by` dict)